bbq/vm/vm.go

```diff
@@ -906,16 +911,37 @@ func opGetMethod(vm *VM, ins opcode.InstructionGetMethod) {
 	if val, ok := receiver.(*interpreter.EphemeralReferenceValue); ok {
 		if refValue, ok := val.Value.(*interpreter.CompositeValue); ok {
 			if refValue.Kind == common.CompositeKindAttachment {
-				base = refValue.GetBaseValue(
-					vm.context,
-					interpreter.ConvertSemaAccessToStaticAuthorization(
-						vm.context,
-						sema.UnauthorizedAccess,
-					),
-					interpreter.EmptyLocationRange,
-				)
-			}
+				// CompositeValue.GetMethod, as in the interpreter we need an authorized reference to base
```

Comment:
It looks like this code is the same as above, maybe consider refactoring these into a common method